src/components/d3/createDonutChart.js

```diff
@@ -3,5 +3,32 @@ import { adjustViewport } from '../../util/chart'
 import { margin } from '../../constants/chartConstants'
 
 function createDonutChart ({ data, width, height, el, tip }) {
+  const [aWidth, aHeight] = adjustViewport(width, height, margin)
 
-}
\ No newline at end of file
+  const radius = Math.min(aWidth, aHeight) / 2
+  const arc = d3.arc().innerRadius(radius * 0.67).outerRadius(radius - 1)
```

Comment:
Why 0.67? Is this a constant?

Reply:
Good point. Those constants (the 0.67 and the 1) control the thickness of the donut. Should I assign them to variables? Not sure what I would call them.

Reply:
Maybe `thicknessMultiplier` for `0.67`? Not sure either haha.

Reply:
Ya that could work. I think the `radius - 1` also controls the thickness too. One basically controls the radius of the inner circle, and the other controls the outer circle.